src/routes/_surfaces/workspace/explore/leaderboards/Leaderboard.svelte

```diff
@@ -69,17 +65,18 @@
   $: atLeastOneActive = !!activeValues?.length;
 
   // Svelte-Query for getting top list start
-  let topListKey = getMetricViewTopListQueryKey(metricsDefId, dimensionId);
-  const topListQuery = useQuery<MetricViewTopListResponse>(topListKey, () =>
-    getMetricViewTopList(
-      metricsDefId,
-      dimensionId,
-      getTopListRequest(metricsExplorer)
-    )
-  );
-  let values: Array<unknown>;
-  $: if ($topListQuery.data) values = $topListQuery.data.data;
-  // Svelte-Query for getting top list end
+  $: topListQuery = useGetMetricViewTopList(metricsDefId, dimensionId, {
```

Comment:
Add a `if(metricsExplorer)` around this. Better to fix here than in a separate PR.